### ISSUE TITLE
Enhance documentation

### ISSUE THREAD

**Issue by rannieo** [einnar82/mpl-ph-api#5]
I'm a believer of  "Great product comes with great documentation."

Create a straightforward documentation for the API.

**Comment by kumaranshu72** [comment]
Can I pick this up?

**Comment by rannieo** [comment]
Yes you can, @kumaranshu72 . Currently the API is on development stage. Better to wait for the other PRs regarding the api endpoints. 

Thanks

**Comment by kumaranshu72** [comment]
@einnar82 In the meanwhile I can add up swagger. I will add Js-docs so that anyone can add the comments and documentation will be self-generated. Let me know if I can go ahead with it.

**Comment by rannieo** [comment]
Thank you for the suggestion,

For the meantime. You may document about the development setup like installation process etc. 😉

**Comment by sauravsolanki** [comment]
@einnar82 I am worked on swagger for around 5 months.. Can I add here? It will pretty good to start initially with Swagger and add  in contribution guides and let PRs member know how to add your API in swagger.

**Comment by sauravsolanki** [comment]
Please assign me this @einnar82
